src/TwigComponent/src/DependencyInjection/TwigComponentExtension.php

```diff
@@ -217,6 +229,10 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->info('Enables the profiler for Twig Component (in debug mode)')
                     ->defaultValue('%kernel.debug%')
                 ->end()
+                ->booleanNode('short_syntax')
```

Comment:
```suggestion
                ->booleanNode('short_tags')
```

More accurate than "syntax", which was a bit of a stretch (same thing to do in the otheer changes -- open to another naming idea)